packages/api/portfolioDrafts/application/createApplicationStep.test.ts

```diff
@@ -154,45 +207,94 @@ describe("Individual Application input validation tests", function () {
   const tooShortName = "abc";
   const tooLongName =
     "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut eleifend lectus ut luctus ultricies nisi.";
-  it("should return error map entries when given Application has a name that is too short", () => {
-    const errors = performDataValidationOnApplication(mockApplicationStepsBadData[0].applications[0]);
+  const tooLongDisplayName =
+    "waaaaaaaaaaaaaaaaaaaaaaaaayyyyyy tooooooooooooooooooooooooooooooooooooo loooooooooonnnnnnnnnnngggggggggg";
+  it("should return error map entries when an operator has a name that is too short", () => {
```

Comment:
```suggestion
  it("should return error map entries when an operator has a name that is too short", async () => {
```

Can we be consistent in this on all our tests? Others here do it, so let's do it for new tests too. If these tests need to be an exception for one reason or another, let's document why.

Reply:
Added `async` to tests.